torchtune/training/__init__.py

```diff
@@ -113,12 +108,7 @@
     "set_torch_num_threads",
     "shard_model",
     "get_shard_conditions",
-    "prepare_model_for_fsdp_with_meta_device",
     "validate_no_params_on_meta_device",
-    "contains_fsdp",
-    "FSDPPolicyType",
-    "get_full_finetune_fsdp_wrap_policy",
```

Comment:
Similar to my comment above, I think this will also need to be removed from the `torchtune.training` API ref doc.

Reply:
There's also a reference to this in the QAT recipe [in the note just below this heading](https://pytorch.org/torchtune/main/tutorials/qat_finetune.html#qat-finetuning-recipe-in-torchtune). Should this note just be removed?

Reply:
Sorry started writing out a comment to this but I guess I never hit send. Yes we should remove the note about `memory_efficient_fsdp_wrap`. You can leave the field in any QAT configs as that'll get handled in #1854 but you can remove from e.g. [here](https://github.com/pytorch/torchtune/blob/d37c6aae9f1da5d1cdebfde02ac846c57b7677d7/recipes/configs/dev/8B_full_experimental.yaml#L71).